Tribler/Core/Upgrade/upgrade.py

```diff
@@ -18,10 +20,13 @@ def __init__(self, session):
         self.notified = False
         self.is_done = False
         self.failed = True
-        self.finished_deferred = None
-        self.pony_upgrader = None
 
         self.current_status = u"Initializing"
+        self._dtp72 = None
```

Comment:
I can infer from the name what the variable does, but overall I think it can be improved. Suggestion: `self._dispersy_to_pony_migrator`?

Reply:
... and then some nice-looking single-liners suddenly become two-liners. One can infer from the name what it is, all of its usages fit on a single screen, it is only used locally and making it longer will result in some line wraps.  I'll leave it as it is.